PerfNext/public/lib/js/util.js

```diff
@@ -733,7 +754,7 @@ function populateHWSpecificArguments(name, variant) {
 		var affinityCapabilityName = chosenNumCores+'way';
 		console.log("util.js: populateMachineSpecificArguments() chosenNumCores: "+chosenNumCores); 
 
-		insertCapabilityEnvVars(affinityCapabilityName, hwEnvClassName, idToFind);
+		insertEnvVar(hwEnvClassName, idToFind, 'NUM_THREADS', chosenNumCores);
```

Comment:
Could you please use `insertEnvVar` to add `CPU_AFFINITY` and get rid of `insertCPUAffinity`?